src/renderer/components/_molecules/data-types/enumerated/table/index.tsx

```
@@ -134,6 +135,7 @@ const EnumeratedTable = ({
         updateDatatype(name, optionalSchema as PLCDataType)
         return newRows
       }
+      handleFileAndWorkspaceSavedState(editor.meta.name)
```

Biome (2.1.2)

[error] This code will never be reached ...

... because either this statement, ...

... this statement, ...

... this statement, ...

... this statement, ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)